### PR TITLE
Add a hash function <select> choice with responsive UI

### PR DESCRIPTION
**Pull request by kota-yata**
Stealing from #462

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -32,12 +32,21 @@ function resetInterface() {
   document.getElementById("sriSnippet").classList.remove('is-active');
 }
 
-async function hashText(message) {
+function digestName(hashAlgorithm) {
+  switch (hashAlgorithm) {
+    case "sha256": return "SHA-256";
+    case "sha384": return "SHA-384";
+    case "sha512": return "SHA-512";
+    default: throw new Error(`invalid hashing algorithm: ${hashAlgorithm}`);
```
Comment (mozfreddyb) [inline app.js]: I think it would be great to default to sha384 for invalid inputs.

**Review by mozfreddyb — APPROVED** [review]
looks good. can you publish this on your github pages for previewing?

**Comment by mozfreddyb** [comment]
oh and please fix the lint violations :-)

**Comment by kota-yata** [comment]
Here's preview page -> https://kota-yata.github.io/srihash.org/

I removed engine field from package.json since it caused the lint violations

**Comment by kota-yata** [comment]
@mozfreddyb

File: package.json [hunk 2]
```diff
@@ -24,9 +24,6 @@
     "glob": "^7.1.7",
     "npm-run-all": "^4.1.5",
     "stylelint": "^13.13.1",
-    "stylelint-config-standard": "^22.0.0"
-  },
-  "engines": {
-    "node": "10.x"
+    "stylelint-config-standard": "^21.0.0"
```
Comment (mozfreddyb) [inline package.json]: Why are you switching from stylelint 22 back to stylelint 21 here?

**Review by mozfreddyb — CHANGES_REQUESTED** [review]
Looks good, but pls undo the downgrade of stylelint-config-standard.

**Review by mozfreddyb — APPROVED** [review]

**Comment by mozfreddyb** [comment]
thank you @kota-yata!

**Pull request merged**